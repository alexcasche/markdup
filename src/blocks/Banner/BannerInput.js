import styled from 'styled-components'
import { tint } from 'polished'

const BannerInput = styled.input`
  width: 100%;
  margin-top: 6px;
  padding: 0;
  background-color: transparent;
  border: none;
  outline: none;
  color: ${props => tint(0.95, props.theme.colors.grey)};
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  font-weight: 400;
  font-size: 20px;
`

export default BannerInput
