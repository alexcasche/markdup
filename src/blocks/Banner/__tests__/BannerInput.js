import BannerInput from '../BannerInput'

describe('BannerInput Element', () => {
  it('renders correctly', () => {
    const tree = render(
      <BannerInput id="Banner-Input" placeholder="Untitled" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
