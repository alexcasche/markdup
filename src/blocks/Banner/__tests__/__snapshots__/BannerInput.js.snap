// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerInput Element renders correctly 1`] = `
.c0 {
  width: 100%;
  margin-top: 6px;
  padding: 0;
  background-color: transparent;
  border: none;
  outline: none;
  color: #afb1b6;
  font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-size: 20px;
}

<input
  className="c0"
  id="Banner-Input"
  placeholder="Untitled"
/>
`;
